lib/Conversion/TorchToTcp/TcpCustomOp.cpp

```diff
@@ -211,6 +214,118 @@ class ConvertAtenFakeQuantizePerChannelAffineOp
+  }
+};
+
+class ConvertAtenMinDimOp : public OpConversionPattern<AtenMinDimOp> {
+public:
+  using OpConversionPattern::OpConversionPattern;
+
+  LogicalResult
+  matchAndRewrite(AtenMinDimOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    torch_to_tcp::TorchToTcpCustomOpConversionHelper helper{op, rewriter,
+                                                            getTypeConverter()};
+    helper.addOperand("self", adaptor.getSelf());
+
+    helper.addIntAttr("dim", op.getDim());
+    helper.addBoolAttr("keepdim", op.getKeepdim());
+
+    return helper.replace();
+  }
+};
+
+class ConvertAtenViewOp : public OpConversionPattern<AtenViewOp> {
+public:
+  using OpConversionPattern::OpConversionPattern;
+
+  LogicalResult
+  matchAndRewrite(AtenViewOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    torch_to_tcp::TorchToTcpCustomOpConversionHelper helper{op, rewriter,
+                                                            getTypeConverter()};
+    Value self = adaptor.getSelf();
+    auto srcType = self.getType().cast<RankedTensorType>();
+    auto resultType =
+        getTypeConverter()->convertType(op.getType()).cast<RankedTensorType>();
+
+    SmallVector<int64_t> size;
+    if (matchPattern(op.getSize(), m_TorchListOfConstantInts(size)) &&
+        srcType.hasStaticShape() && resultType.hasStaticShape())
+      return rewriter.notifyMatchFailure(op, "only dynamic shape is supported");
+
+    helper.addOperand("self", self);
+    Operation *primListOp = op.getSize().getDefiningOp();
+    auto listConstruct = dyn_cast<Torch::PrimListConstructOp>(primListOp);
+    if (!listConstruct) {
+      return rewriter.notifyMatchFailure(
+          op, "Size must come from PrimListConstructOp");
+    }
+    int idx = 0;
+    for (Value value : listConstruct.getElements()) {
+      int64_t dimSize;
+      if (!matchPattern(value, m_TorchConstantInt(&dimSize))) {
+        size.push_back(ShapedType::kDynamic);
+        if (!isa<TorchConversion::FromI64Op>(value.getDefiningOp()))
+          return rewriter.notifyMatchFailure(
+              op, "dynamic dim size should come from FromI64Op");
+        auto conversionOp =
+            dyn_cast<TorchConversion::FromI64Op>(value.getDefiningOp());
+        if (!isa<arith::IndexCastOp>(conversionOp.getOperand().getDefiningOp()))
+          return rewriter.notifyMatchFailure(
+              op, "dynamic dim size should come from IndexCastOp");
+        auto indexCastOp = dyn_cast<arith::IndexCastOp>(
+            conversionOp.getOperand().getDefiningOp());
+        if (!isa<tensor::DimOp>(indexCastOp.getIn().getDefiningOp()))
+          return rewriter.notifyMatchFailure(
+              op, "dynamic dim size should come from DimOp");
+        auto dimOp =
+            dyn_cast<tensor::DimOp>(indexCastOp.getIn().getDefiningOp());
+        helper.addOperand("idx_" + std::to_string(idx), dimOp);
```

Comment:
I see you are matching a specific pattern. Can you add a comment to show the pattern? That would make it easier to read the code.